sdk/README.md

```diff
@@ -81,3 +81,16 @@ Here we update the `Compiler` of the KFP SDK to generate `Tekton` YAML for a bas
 ## Test Kubeflow Pipelines with Tekton
 
 Please [refer to the instructions here](./python/tests/README.md) as you work on a PR test sample Kubeflow Pipelines in their test data folder to ensure your PR is improving the number of successful samples
+
+## Compile Kubeflow Pipelines as Tekton pipelineRun
+
+Compiling Kubeflow Pipelines into Tekton pipelineRun is currently under the experimental stage. Below are the features that only supported within pipelineRun
+- Affinity
+- Node Selector
+- Tolerations
```

Comment:
How many total features are there, can we add here?

Reply:
Right now these are the only three features supported in pipelineRun. There will be more coming in the future RPs.

Reply:
yeah - but how many total are there?

Reply:
I added a link pointing to the list of pipelinerun features.